Libraries/LibWeb/DOM/Node.cpp

```diff
@@ -2532,7 +2532,7 @@ ErrorOr<String> Node::name_or_description(NameOrDescription target, Document con
         //
         // AD-HOC: We’ve reordered substeps C and D from https://w3c.github.io/accname/#step2 — because
         // the more-specific per-HTML-element requirements at https://w3c.github.io/html-aam/#accname-computation
-        // necessitate doing so, and the “input with label for association is superceded by aria-label” subtest at
+        // necessitate doing so, and the “input with label for association is superseded by aria-label” subtest at
```

Comment:
This quotes the name of a WPT test so should be left as it was.

Reply:
Yes sir 🫡